app/views/content_items/how_government_works.html.erb

```diff
@@ -0,0 +1,552 @@
+            } %>
+
+            <p class="govuk-body">These are part of government departments and usually provide government services rather than decide policy - which is done by the department that oversees the agency.</p>
+            <p class="govuk-body">An example is the <%= link_to "Driver and Vehicle Licensing Agency", "/government/organisations/driver-and-vehicle-licensing-agency", class: "govuk-link" %> (overseen by the <%= link_to "Department for Transport", "/government/organisations/department-for-transport", class: "govuk-link" %>).</p>
+
+            <%= render "govuk_publishing_components/components/heading", {
+              text: "Other public bodies",
+              font_size: "m",
+              heading_level: 3,
+              margin_bottom: 4,
+            } %>
+
+            <p class="govuk-body">These have varying degrees of independence but are directly accountable to ministers. There are 4 types of non-departmental public bodies (NDPBs).</p>
+            <p class="govuk-body">Executive NDPBs do work for the government in specific areas - for example, the Environment Agency.</p>
+            <p class="govuk-body">Advisory NDPBs provide independent, expert advice to ministers - for example, the Committee on Standards in Public Life.</p>
+            <p class="govuk-body">Tribunal NDPBs are part of the justice system and have jurisdiction over a specific area of law - for example, the Competition Appeal Tribunal.</p>
+            <p class="govuk-body">Independent monitoring boards are responsible for the running of prisons and treatment of prisoners - for example, His Majesty's Inspectorate of Prisons.</p>
+          </div>
+        </div>
+
+        <% if !@content_item.reshuffle_in_progress? %>
+          <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+              <div class="depts">
+                <div class="depts__item govuk-!-display-inline-block">
+                  <%= render "govuk_publishing_components/components/big_number", {
+                    number: @content_item.department_counts["ministerial_departments"],
+                    label: "Ministerial departments",
+                    href: "/government/organisations#departments"
+                  } %>
+                </div>
+                <div class="depts__item govuk-!-display-inline-block">
+                  <%= render "govuk_publishing_components/components/big_number", {
+                    number: @content_item.department_counts["non_ministerial_departments"],
+                    label: "Non-ministerial departments",
+                    href: "/government/organisations#non_ministerial_departments"
+                  } %>
+                </div>
+                <div class="depts__item govuk-!-display-inline-block">
+                  <%= render "govuk_publishing_components/components/big_number", {
+                    number: "#{@content_item.department_counts["agencies_and_public_bodies"].floor(-2)}+",
```

Comment:
Could this go into the presenter too?

Reply:
I'll move the logic for this into the presenter.